mpy-cross/qstrdefsport.h

```diff
@@ -1 +1,3 @@
 // qstrs specific to this port
+// *FORMAT-OFF*
+Q(<stdin>)
```

Comment:
This shouldn't be needed because it's in `py/qstrdefs.h`.

Reply:
It didn't work for me when I tried it (it expanded to `_lt_stdin_gt_`), having it specified twice shouldn't hurt, right?

Reply:
I just tried it without this line added, and it works for me.  Builds without error and then running it looks correct:
```
$ echo -n | ./build/mpy-cross - | xxd 
00000000: 4d06 001f 0200 0e3c 7374 6469 6e3e 000f  M......<stdin>..
00000010: 2800 0201 5163                           (...Qc
```

Reply:
Alright, not sure what that was then, I've removed it.